### DISCUSSION TITLE
How to force empty trash?

### DISCUSSION THREAD

**Discussion by doongjohn** [leo-arch/clifm#301] [Q&A] [answered]
Is there a way to ignore remove write-protected prompt when clearing the trash via `t empty`?
![image](https://github.com/user-attachments/assets/d05b8f9c-69fe-42db-9129-4208e82945e4)

**Comment by leo-arch** [comment, marked as answer]
Hi @doongjohn.

When **clifm** trashes a file, it moves it to the trash directory without changing any permissions. So, if the file was write-protected before trashing it, it will be write-protected after as well.

However, I guess I can force the remove command to not prompt for confirmation (like `rm -f`). It makes sense, since a confirmation prompt is displayed immediately before anyway.

If there isn't any drawback, it will make the change.

EDIT: It should work as expected since the [last commit](https://github.com/leo-arch/clifm/commit/2d6d7ac33f6944c4e10dc8cf96f3e8c0859a39eb).
